docassemble/MACourts/macourts.py

```diff
@@ -421,12 +421,14 @@ def load_courts_from_file(self, court_name, data_path='docassemble.MACourts:data
 
         path = path_and_mimetype(os.path.join(data_path,json_path+'.json'))[0]
 
-        with open(path) as courts_json:
+        with open(path,encoding='utf-8-sig') as courts_json:
```

Comment:
I think this change is safe. But we should avoid changing unrelated code when possible. Alternative is to remove BOM from the JSON files as that does not seem to be part of the spec.

This is the only clue I found about safety, but I'm satisfied: https://github.com/psychopy/psychopy/pull/2168